templates/python/test-requirements.mustache

```diff
@@ -11,10 +11,11 @@ py>=1.4.31
 randomize>=0.13
 tox<4.0.0
 
+# Build package
```

Comment:
**explain:** Moved it here, from being a comment in the `Makefile`, so that it's installed as other requirements are